daml-foundations/daml-tools/daml-extension/src/extension.ts

```diff
@@ -453,26 +441,6 @@ const telemetryConsentKey = 'telemetry-consent'
 const privacyPolicy = 'https://www.digitalasset.com/privacy-policy'
 
 
-// remove this once env.openExternal has been around for 6 months (1st July 2019)
```

Comment:
The date hasn’t passed but we already require VSCode `1.32.0` explicitely in `package.json` so there is no point in keeping this around.